src/CommonLib/Processors/PortScanner.cs

```diff
@@ -37,28 +38,27 @@ public virtual async Task<bool> CheckPort(string hostname, int port = 445, int t
 
             if (PortScanCache.TryGetValue(key, out var status))
             {
-                _log.LogTrace("Ping cache hit for {HostName} on {Port}: {Status}", hostname, port, status);
+                _log.LogTrace("Port scan cache hit for {HostName}:{Port}: {Status}", hostname, port, status);
                 return status;
             }
 
             try
             {
                 using var client = new TcpClient();
                 var ca = client.ConnectAsync(hostname, port);
-                await Task.WhenAny(ca, Task.Delay(timeout));
-                client.Close();
-                if (!ca.IsFaulted && ca.IsCompleted)
+                if (await Task.WhenAny(ca, Task.Delay(timeout)) == ca)
```

Comment:
I just want to call this out as a sanity check. The PR says that this is just logging changes but this appears to be a logic change. Just making sure it was intentional.